indigo/distribution.yaml

```diff
@@ -3583,6 +3583,26 @@ repositories:
       url: https://github.com/JenniferBuehler/gazebo-pkgs.git
       version: master
     status: developed
+  gcloud_speech:
+    doc:
+      type: git
+      url: https://github.com/CogRob/gcloud_speech.git
+      version: master
+    release:
+      packages:
+      - gcloud_speech
+      - gcloud_speech_msgs
+      - gcloud_speech_utils
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/CogRobRelease/gcloud_speech-release.git
+      version: 0.0.1-0
+    source:
+      test_pull_requests: true
```

Comment:
Have you followed the [instructions](http://wiki.ros.org/buildfarm/Pull%20request%20testing) to provide access to this repo or the org unit to the @ros-pull-request-builder user? If not, either remove this line or provide the necessary access as described in the wiki page.

Reply:
I have just given write permission to "ros-pull-request-builder" and added the webhook. Thanks for pointing out.

Reply:
I see "Awaiting ros-pull-request-builder's response" in my GitHub settings. Is this normal? Does the webhook automatically accept the invitation?

Reply:
No, the invitation has to be manually accepted (which I just did). Since this has been merged before the invitation was accepted the generated PR job won't work until the job is being regenerated in the next night.

Reply:
Thank you very much!